worker/src/websocket.rs

```diff
@@ -225,7 +293,7 @@ pub mod ws_events {
 
     impl MessageEvent {
```

Comment:
for `MessageEvent`'s `get_*` methods, do you think being consistent with `Request` impl naming is helpful? 

e.g. 
```rust
    /// Access this request's body encoded as JSON.
    pub async fn json<B: DeserializeOwned>(&mut self) -> Result<B> { ... }
    /// Access this request's body as plaintext.
    pub async fn text(&mut self) -> Result<String> { ... }
    /// Access this request's body as raw bytes.
    pub async fn bytes(&mut self) -> Result<Vec<u8>> { ... }
```

Reply:
Yeah, that'd probably preferable.